src/react/components/SvgRadioButton.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button, Select, Tooltip } from 'oskari-ui';
+import { Form, Card, Space, Input, Row, Radio } from 'antd';
+import styled from 'styled-components';
+
+const formLayout = {
+    labelCol: { span: 24 },
+    wrapperCol: { span: 24 }
+}
+
+const RadioIcon = styled(Radio.Button)`
+    height: 34px;    
+    margin: 0 7px 10px 0;
+    padding: 0;
+    width: 34px;
+`;
+
+/**
+ * @class SvgRadioButton
+ * @calssdesc <SvgRadioButton>
+ * @memberof module:oskari-ui
+ * @see {@link module:oskari-ui/util.LocaleProvider|LocaleProvider}
+ * @param {Object} props - { }
+ *
+ * @example <caption>Basic usage</caption>
+ * <SvgRadioButton props={{ ...exampleProps }}/>
+ */
+
+export class SvgRadioButton extends React.Component {
```

Comment:
Make this a function component instead.

Reply:
Also naming could be improved. Since this gets options as props this is a selection between those options rather than a single radiobutton option?